meson_options.txt

```diff
@@ -0,0 +1 @@
+option('server', type: 'feature', description: 'Controls whether the CDBA server is built')
```

Comment:
```suggestion
option('server', type: 'feature', description: 'Controls whether the CDBA server is built, default true if dependencies are found on the system')
```

Reply:
I opted for a slightly different text here.